docs/src/cookbooks/cookbook.md

```diff
@@ -1,10 +1,14 @@
 ---
 layout: docs
-title:  "Cookbook"
+title:  "General Cookbooks"
 section: "chisel3"
 ---
 
-Welcome to the Chisel cookbook. This cookbook is still in early stages. If you have any requests or examples to share, please [file an issue](https://github.com/ucb-bar/chisel3/issues/new) and let us know!
+# General Cookbooks
+
+Welcome to the Chisel Cookbooks, where we capture frequently-asked design patterns.
+If you have any requests or examples to share,
+please [file an issue](https://github.com/chipsalliance/chisel3/issues/new) and let us know!
```

Comment:
Should this point at chisel3 or at the website repo?

Reply:
this is what it currently says, and since the content is here it makes sense here I think